Sources/Server/Response.swift

```diff
@@ -72,6 +72,55 @@ public class Response {
         let header = "HTTP/1.1 302\r\nContent-Length: \(body.utf8.count)\r\nContent-type: text/html\r\nLocation: /\r\n\r\n" 
         return header + body
     }
+
+    private func handleForm(request: Request) -> String {
+        let filePath = NSURL.fileURL(withPath: "log.txt")
+        if (request.method == "POST") {
+            writeText(requestBody: request.body, path: filePath)
+        } else {
+            let rawLogData = readText(path: filePath)
+            let updatedData = getUpdatedText(data: rawLogData, bodyText: request.body) 
+            writeText(requestBody: updatedData, path: filePath)     
+        }
+        let body = ""
+        let header = buildHeader(statusCode: status200, contentLength: body.utf8.count)
+        return header + body
+    }
+
+    private func writeText(requestBody: [String: String], path: URL) {
+        let text =  requestBody.map{ "\($0)=\($1)" }.joined(separator:"\n")
+            do {
+                try text.write(to: path, atomically: false,
+                encoding: String.Encoding.utf8)
+            } catch {
+                print("Error: \(error)")
+            }
+    }
+
+    private func readText(path: URL) -> String {
+        var logData: String = String()
+            do {
+                logData = try String(contentsOf: path, encoding: String.Encoding.utf8) 
+            } catch {
+                print("Error: \(error)")
```

Comment:
"Error: " is a very vague prefix, even more so when used in multiple places. Be descriptive to give the best chance at tracking down the cause of the error. Also, you could be logging the errors.